tests/isar/services/readers/test_mission_reader.py

```diff
@@ -94,6 +94,6 @@ def test_thermal_image_task(mission_reader):
 
     assert isinstance(task, TakeThermalImage)
     assert hasattr(task, "target")
-    assert task.task_name == "take_thermal_image"
+    assert task.name == "take_thermal_image"
```

Comment:
Add check for task.id
```suggestion
    assert task.name == "take_thermal_image"
    assert hasattr(task, "id")
```